### PR TITLE
Readability improvements for README.md

### PR DESCRIPTION
**Pull request by ejsears**
## Major Changes in this PR

- Add some requested information to the README

## PR Checklist: (Checked-off items are either done or do not apply to this PR)
 
- [x] I have performed a self-review of my code
- [x] CHANGELOG.md is updated (if end-user facing)
- [x] Sent notification to software-design channel requesting review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,25 +226,33 @@ Debug information about the respooler system can be found by visiting the follow
 
 `{ip address}/printer/objects/query?AFC`
 
-### LEDs not displaying correct color
+<details>
+<summary>LEDs not displaying correct color</summary>
```
Comment (jimmyjon711) [inline README.md:230]: Summary sections look weird and don't apply then code sections and new lines when looking at the rich text
image

Comment (ejsears) [reply README.md:230]: Gah. They showed up fine for me. I'll revert them.

Comment (ejsears) [reply README.md]: @jimmyjon711 Adjusted.

**Review by jimmyjon711 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jimmyjon711) [reply README.md]: Looks good to me approved

**Pull request merged**